webapp/app.py

```diff
@@ -38,6 +38,14 @@ def create_app(testing=False):
 
     if not testing:
         talisker.flask.register(app)
+        import webapp.api.blog
+        import webapp.api.dashboard
+        import webapp.api.sso
+
+        talisker.requests.configure(webapp.api.blog.api_session)
```

Comment:
Same as above - why only add talisker stuff when not testing? It shouldn't break when testing, and keeps your tests closer to production.

Reply:
this would imply a refactor on our test suite since we don't run the tests with talikser. For now I would rather keeping .it like this and maybe work on this on another task